app/components/PurchaseSubscription/index.tsx

```diff
@@ -1,9 +1,82 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useSubscriptions } from '~/hooks/useSubscriptions';
+import Modal from '~/components/Modal';
+import Icon from '~/components/common/Icon';
+import { PrimaryButton } from '~/components/common/Button';
+import { TextLink } from '~/components/common/Link';
+import './purchase.css'
 
-const PurchaseSubscription = () => (
-  <div>
-    <h1 style={{color: 'black'}}>hello</h1>
-  </div>
-);
+const PurchaseSubscription = () => {
+
```

Comment:
Remove the function body since it's not used. the component could simply look like:
```
const PurchaseSubscription = () => (
  <>
     ...
  </>
);
```